### ISSUE TITLE
Use a yellow badge with "?" icon for ignored tests (currently blank)

### ISSUE THREAD

**Issue by dialex** [picklesdoc/pickles#460]
I followed the official doc to [change the SpecRunner report](http://docs.picklesdoc.com/en/latest/IntegratingTestResultsFromSpecRun/).
But what are my next steps?

So I googled and found a [guide on youtube](https://youtu.be/UkoS-KpDi2w?t=4m8s) explaining how integrate the results of a SpecRun execution (pass/fail) into Pickles DHTML... however it used PicklesUI and I need something that will run on CI.

I continued to look for clues and found this `link-results-file` parameter. Why isn't that referenced on the "Integrating Test Results From SpecFlow+ Runner"?

Finally this was my execution, which resulted in a cryptic error:

```
pickles.exe
  --feature-directory=Features  --output-directory=GeneratedDocs\  --documentation-format=dhtml
  --link-results-file=TestResults\Specs_Default_2017-05-17T095431.html  --test-results-format=specrun

Pickles v.2.15.0.0
Generating documentation based on the following parameters
----------------------------------------------------------
Feature Directory              : (...)
Output Directory               : (...)
Project Name                   :
Project Version                :
Language                       : en
Incorporate Test Results?      : Yes
Include Experimental Features? : No
Exclude Tag                    :
Test Result Format        : SpecRun
Test Result File          : (...)TestResults\PartnerPortal.Specs_Default_2017-05-17T095431.html

Pickles did not complete successfully An error occurred during the activation of a particular registration. See the inner exception for details. Registration: Activator = ITestResults (DelegateActivator), Services = [PicklesDoc.Pickles.ObjectModel.ITestResults], Lifetime = Autofac.Core.Lifetime.RootScopeLifetime, Sharing = Shared, Ownership = OwnedByLifetimeScope ---> An error occurred during the activation of a particular registration. See the inner exception for details. Registration: Activator = SpecRunResults (ReflectionActivator), Services = [PicklesDoc.Pickles.TestFrameworks.SpecRun.SpecRunResults], Lifetime = Autofac.Core.Lifetime.CurrentScopeLifetime, Sharing = None, Ownership = OwnedByLifetimeScope ---> An exception was thrown while invoking the constructor 'Void .ctor(PicklesDoc.Pickles.IConfiguration, PicklesDoc.Pickles.TestFrameworks.SpecRun.SpecRunSingleResultLoader, PicklesDoc.Pickles.TestFrameworks.SpecRun.SpecRunScenarioOutlineExampleMatcher)' on type 'SpecRunResults'. ---> StartIndex cannot be less than zero.
Parameter name: startIndex (See inner exception for details.) (See inner exception for details.) (See inner exception for details.)
```

Can you help me?

**Comment by dirkrombauts** [comment]
Regarding the documentation: I'm happy to hear your suggestions to improve it. I'm even happier if you send a pull request on the https://github.com/picklesdoc/docs repository :-)

Regarding your problem: would it be possible for you to send me your feature files and your test result file? This kind of error is very hard to debug without that information.

**Comment by dialex** [comment]
Will do, I've added to my todo list 😉

I've attached a [zip](https://github.com/picklesdoc/pickles/files/1013678/useful.files.zip) with:

- my pickles command
- one of the feature files
- the SpecRun generated html with the test results
- and the generated Pickles report

My latest attempt didn't produce the error but I did not see the results on Pickles' report.
Is there any step by step instructions for SpecRun html files (not *Unit xml files)? Thanks for your help.

**Comment by dialex** [comment]
Will do, I've added to my todo list 😉

I've attached a [zip](https://github.com/picklesdoc/pickles/files/1013899/files.zip) with:

- my pickles command
- one of the feature files
- the SpecRun html with the test results + template file
- and the generated Pickles report

The original SpecFlow report:

![image](https://cloud.githubusercontent.com/assets/2140450/26244471/aac8f8aa-3c87-11e7-8233-ec1f5c631152.png)

I managed to get "some" integration... however results are not coherent. As you can see two of the features have no results. When you click it, one of them is passed and the other has nothing:

![image](https://cloud.githubusercontent.com/assets/2140450/26244530/e8205f0e-3c87-11e7-9ee5-39cedb949f88.png)

What's wrong?

**Comment by dialex** [comment]
OK, I think I got it, it's not a bug @dirkrombauts but it's definitely something to improve.

When a test is ignored it doesn't show any icon, neither green nor red, it shows "blank". This lack of detail might confuse the reader, as the reader has no idea what's wrong with that test. That explains the "blank" for the second test on the right side.

On the left side, we also have "blank". That's because we have 1 green (passed) and 1 "blank" (ignored). It seems as soon as one "blank" occurs, the aggregate result displayed on the left side is also "blank".

---------

**I suggest the "blank" test result for ignored tests is replaced with a yellow badge with a question mark icon**.

**Comment by dirkrombauts** [comment]
Hi,

Thanks for your contribution to the docs, and your suggestion for the app. I hope I'll be able to put in a bit more work on Pickles in the couple of days to include this.

**Comment by dirkrombauts** [comment]
Fixed by #463

**Comment by dirkrombauts** [comment]
Release as part of version [2.16.0](https://github.com/picklesdoc/pickles/releases/tag/v2.16.0).
